docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/terraform-setup.md

```diff
@@ -0,0 +1,285 @@
+
+:::tip
+
+If you are completely new to Terraform and the idea of IaC, read through the [Terraform IaC documentation](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/infrastructure-as-code) and give their [interactive quick start](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/infrastructure-as-code#quick-start) a try for a basic understanding.
+
+:::
+
+## Prerequisites
+
+- An [AWS account](https://docs.aws.amazon.com/accounts/latest/reference/accounts-welcome.html) to create any resources within AWS.
+- [Terraform (1.6.x)](https://developer.hashicorp.com/terraform/downloads)
+- [Kubectl (1.28.x)](https://kubernetes.io/docs/tasks/tools/#kubectl) to interact with the cluster.
+- [IAM Roles for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) (IRSA) configured.
+  - This simplifies the setup by not relying on explicit credentials and instead creating a mapping between IAM roles and Kubernetes service account based on a trust relationship. A [blog post](https://aws.amazon.com/blogs/containers/diving-into-iam-roles-for-service-accounts/) by AWS visualizes this on a technical level.
+  - This allows a Kubernetes service account to temporarily impersonate an AWS IAM role to interact with AWS services like S3, RDS, or Route53 without having to supply explicit credentials.
+
+## Considerations
+
+This is a basic setup to get started with Camunda 8 but does not reflect a high performance setup. It's a good starting point to get ready for production by utilizing [IaC tooling](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/infrastructure-as-code).
+
+Terraform can be opaque in the beginning. If you solely want to get an understanding for what is happening, you may try out the [eksctl guide](./eksctl.md) to understand what resources are created and how they interact with each other.
+
+To try out Camunda 8 or develop against it, consider having a look at our [SaaS offering](https://camunda.com/platform/). If you already have an AWS EKS cluster, consider skipping to the [Helm guide](./eks-helm.md).
+
+For the simplicity of this guide, certain best practices will be provided with links to additional documents, enabling you to explore the topic in more detail.
+
+:::warning
+Following this guide will incur costs on your Cloud provider account. Those being namely for the managed Kubernetes (EKS) service, resulting Kubernetes nodes in EC2, Elastic Block Storage (EBS), and Route53. More information can be found on [AWS](https://aws.amazon.com/eks/pricing/) and their [pricing calculator](https://calculator.aws/#/) as the total cost varies per region.
+:::
+
+## Outcome
+
+Following this tutorial and steps will result in:
+
+- An AWS EKS 1.28 Kubernetes cluster with four nodes ready for Camunda 8 installation.
```

Comment:
are we fixing to 1.28? Should we future-proof this and install the latest available?

Reply:
changed it for Terraform as we control it better via an external module.
Kept it for `eksctl` as it's a snapshot in time.